timeless/sync/sync.py

```diff
@@ -1,13 +1,35 @@
+from timeless.restaurants.models import Location
 """
     Syncs Poster with database.
-    @todo #66:30min Implement synchronization between Poster and Databse for
+    @todo #88:30min Implement synchronization between Poster and Databse for
      Locations. Data coming from Poster has priority upon data stored in our
-     database. After implementing this, remove @unittest.skip annotation from
-     test_sync_location in test_sync.py. Do not forget to create cron script and
-     put int into scripts folder to make this sync periodic.
+     database. Create integration tests for database layer executing 
+     sync_location passing an database backed company object. Suggestion: 
+     start by creating structure for integration tests with database
 
 """
 class PosterSync():
 
     def sync_location(poster, company):
-        raise Exception("sync_location not implemented yet")
+        synced =[]
+        for location in company.locations:
+            for loc in poster.locations():
+                if location.id == loc["id"]:
+                    synced.append(
+                        Location(
+                            id=location.id,
+                            name=loc["name"],
+                            code=loc["code"],
+                            company_id=location.company_id,
+                            country=loc["country"],
+                            region=loc["region"],
+                            city=loc["city"],
+                            address=loc["address"],
+                            longitude=loc["longitude"],
+                            latitude=loc["latitude"],
+                            type=loc["type"],
+                            status=loc["status"],
+                            comment=loc["comment"]
+                        )
+                    )
+        company.locations = synced
```

Comment:
@paulodamaso Why you didn't save it to database?

Reply:
@vladarefiev I've kept the sync logic here, decoupled from data storage. The tests in `test_sync.py` where made decoupled from database, and to test them against it I would have to do a lot of mocks and / or create all the necessary data demanded by `Location` table, and create an integration testing logic for the project. This way we test if the sync routine is syncing the object, and we can make only the integration test later.

Reply:
@paulodamaso got it. thanks